redaxo/src/addons/phpmailer/lib/mailer.php

```diff
@@ -104,6 +104,8 @@ public function send()
         return rex_timer::measure(__METHOD__, function () {
             $addon = rex_addon::get('phpmailer');
 
+            rex_extension::registerPoint(new rex_extension_point('PHPMAILER_PRESEND', $this));
```

Comment:
Analog zum bereits existierenden EP `ART_PRE_DELETED` würde ich die Worte trennen.

 ```suggestion
            rex_extension::registerPoint(new rex_extension_point('PHPMAILER_PRE_SEND', $this));
```

Reply:
Super Danke Passt
https://github.com/redaxo/redaxo/commit/508e46b7cbb79adc1fc82773d346f9408ccbd505